discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -341,49 +346,50 @@ def wait_for_pending_for_input_status(self):
             timeout=consts.PENDING_USER_ACTION_TIMEOUT
         )
 
-    def wait_for_at_least_one_host_to_boot_during_install(self, nodes_count=1):
+    def wait_for_at_least_one_host_to_boot_during_install(self):
         utils.wait_till_at_least_one_host_is_in_stage(
             client=self.api_client,
             cluster_id=self.id,
             stages=[consts.HostsProgressStages.REBOOTING],
-            nodes_count=nodes_count
+            nodes_count=self._config.nodes_count
         )
 
-    def wait_for_non_bootstrap_masters_to_reach_configuring_state_during_install(self):
+    def wait_for_non_bootstrap_masters_to_reach_configuring_state_during_install(self, num_masters: int = None):
```

Comment:
most of the functions in this class doesn't set expected type for env, so can we have standart here? or we change all or lets leave the way it is now.

Reply:
it can be done with another PR

Reply:
@tsorya Already pushed the fix